schema/vod-source.yaml

```diff
@@ -159,3 +161,48 @@ $defs:
           Additional information: Semicolon-separated series of additional
           information fields.
     required: ['genome_assembly', 'chrom', 'pos', 'ref', 'alt']
+  
+  LocationDescriptor:
+    description: >-
+      This descriptor is intended to reference :ref:`VRSLocation` value objects. In addition to the
+      attributes inherited from its :ref:`ValueObjectDescriptor` parent class, the Sequence Location
+      Descriptor has the following attributes:
+    additionalProperties: false
+    type: object
+    properties:
+      type:
+        type: string
+        enum: [“LocationDescriptor”]
+        description: >-
+          MUST be “LocationDescriptor”
+      location_id:
+        $ref_curie: "vrs:CURIE"
+        description: >-
+          This MUST be provided if `location` is omitted
+      location:
+        $ref: "#/$defs/VRSLocation"
```

Comment:
This requires a `$ref_curie` style reference; see line 179 above for an example.